data/archiver/dataclass.py

```diff
@@ -46,6 +46,14 @@ def __init__(self, uuid, file_name, cloud_url, size=0, compressed=False, md5=Non
         self.success = success
         self.error = error
 
+    @classmethod
+    def from_file(cls, file):
+        return cls(file["uuid"], file["file_name"], file["cloud_url"])
+
+    @classmethod
+    def not_found_error(cls, uuid):
+        return cls(uuid, None, None, success=False, error="File not found in Ingest.")
```

Comment:
If you are keeping the None args, at least used the named arguments invocation, so the reader knows what they mean.